docs/dev/connectors/kafka.md

```diff
@@ -804,4 +776,36 @@ When using standalone Flink deployment, you can also use `SASL_SSL`; please see
 For more information on Flink configuration for Kerberos security, please see [here]({{ site.baseurl}}/ops/config.html).
 You can also find [here]({{ site.baseurl}}/ops/security-kerberos.html) further details on how Flink internally setups Kerberos-based security.
 
+## Troubleshooting
+
+<div class="alert alert-warning">
+If you have a problem with Kafka when using Flink, keep in mind that Flink only wraps <tt>KafkaConsumer</tt> or <tt>KafkaProducer</tt>
```

Comment:
Can we include links to the Javadocs for `KafkaConsumer` and `KafkaProducer`?

Reply:
Moreover, the connectors for 0.8 and 0.9+ uses different Kafka Java APIs. Might want to point that out.

Reply:
That would be an external link to Kafka docs (for what version?). 

What do you mean different Kafka Java APIs? (I'm/I wasn't aware of that, so I don't know what should I point out)

Reply:
Kafka 0.8 uses a lower-level client called `SimpleConsumer`: https://www.javadoc.io/doc/org.apache.kafka/kafka_2.10/0.8.0

Other versions use a higher level client, called the `KafkaConsumer`:
https://kafka.apache.org/10/javadoc/?org/apache/kafka/clients/consumer/KafkaConsumer.html

Reply:
Ok, done. Thanks for the review @tzulitai.